ios/ImageCropPicker.m

```diff
@@ -458,31 +492,46 @@ - (void)qb_imagePickerController:
                          
                          dispatch_async(dispatch_get_main_queue(), ^{
                              [lock lock];
+                             
                              @autoreleasepool {
                                  UIImage *imgT = [UIImage imageWithData:imageData];
                                  UIImage *imageT = [imgT fixOrientation];
                                  
-                                 ImageResult *imageResult = [self.compression compressImage:imageT withOptions:self.options];
-                                 NSString *filePath = [self persistFile:imageResult.data];
-                                 
-                                 if (filePath == nil) {
-                                     [indicatorView stopAnimating];
-                                     [overlayView removeFromSuperview];
-                                     [imagePickerController dismissViewControllerAnimated:YES completion:[self waitAnimationEnd:^{
-                                         self.reject(ERROR_CANNOT_SAVE_IMAGE_KEY, ERROR_CANNOT_SAVE_IMAGE_MSG, nil);
-                                     }]];
-                                     return;
+                                 if ([self.options[@"compressImageMaxWidth"] integerValue] == 0 && [self.options[@"compressImageMaxHeight"] integerValue] == 0 && [self.options[@"compressImageQuality"] isEqualToNumber:@1]) {
+                                     
+                                     [selections addObject:[self createAttachmentResponse:@""
+                                                                      withLocalIdentifier:phAsset.localIdentifier
+                                                                             withFilename:sourceURL.lastPathComponent
+                                                                                withWidth:[NSNumber numberWithFloat:imageT.size.width]
+                                                                               withHeight:[NSNumber numberWithFloat:imageT.size.height]
+                                                                                 withMime:[ImageCropPicker contentTypeForImageData:imageData]
```

Comment:
is there a way to avoid our custom function to get a mime type?

Reply:
ImagePicker.openPicker({
      mediaType:'photo'  <==Can be 'video'or 'any'
    })
    .then(file => {
      console.log(file.mime);  <== This shall give mime type of the media chosen
    })